src/patternfly/components/Progress/progress.scss

```diff
@@ -126,14 +130,16 @@
 
 // the progress__description is displayed above the progress bar on the left
 .pf-c-progress__description {
+  word-break: break-word;
```

Comment:
should we have an option to truncate it ? @mcarrano

Reply:
ping @mcarrano @mceledonia

Reply:
Yes, I think it would make sense to offer that option, if possible.

Reply:
@christiemolloy @mcarrano added truncation